content/metadata.yaml

```diff
@@ -511,6 +511,8 @@ authors:
     name: Yael Evelyn Marshall
     initials: YEM
     orcid: 0000-0003-3723-9683
+    affiliations:
+      - Independent Researcher
```

Comment:
@rando2 I think you didn't see my comment for what to write for affiliations. For some reason I don't see an option to edit this file, so please put under affiliations these 2 things (& remove "independent researcher"):
  - freelance copyeditor
  - recent PhD, Princeton University

Reply:
@rando2 I put in the affiliations myself through PR #546 so please remove this file from this PR but keep the acknowledgments file above it & use the other PR to update the affiliations correctly. thanks :-) I was trying to make things less messy not more messy but as long as it gets done :-)